addons/knowledge/static/src/js/knowledge_controller.js

```diff
@@ -6,14 +6,15 @@ import FormController from 'web.FormController';
 import { MoveArticleToDialog } from './widgets/knowledge_dialogs.js';
 import emojis from '@mail/js/emojis';
 
-const emojisBlacklist = ['💩', '👎', '💔', '😭', '😢', '😐', '😕', '😞', '😢'];
+const emojisBlacklist = ['💩', '👎', '💔', '😭', '😢', '😐', '😕', '😞', '😢', '💀'];
```

Comment:
I would remove the blacklist (that should be called exclusion list) and move into a source list. Put in that one a few of "cool" emojis and this should be good enough.

Reply:
It looks like after the long discussion we've had on Discord, we'll keep the blacklist (to be renamed to `disallowedEmojis`) for now